app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -916,6 +932,33 @@ impl Node {
             color::Lcha::transparent()
         }
     }
+
+    fn node_bounding_box(
+        node_size: Vector2,
+        node_position: Vector2,
+        visualization_enabled_and_visible: bool,
+        visualization_size: Vector2,
+    ) -> BoundingBox {
+        let position_of_bounding_box_of_node = node_position - Vector2(0.0, node_size.y / 2.0);
```

Comment:
This computation bases on some assumptions of how nodes are aligned now. There is somewhere a code which aligns the nodes in this way. If that code changes, this would need to be changed too, which makes it hard to refactor. In the place where the alignment of node is computed, there should be a function exposing this information and it should be used here instead of duplicating this logic.